Scripts/PAIF-N/04-enable-wcp.ps1

```diff
@@ -0,0 +1,284 @@
+
+$domainName = 'sfo-w01'
+
+$dnsServer = '10.0.0.250'
+$ntpServer = '10.0.0.250'
+
+$domain = 'vrack.vsphere.local'
+
+# --------------------------------------------------------------------------------------------------------------------------
+# Workload domain parameters - stripped down version of $domainSpec from 01-deploy-vcf-workload-domain.ps1
+$domainSpec = @{
+   VCenterSpec = @{
+      RootPassword = "VMware123!"
+      NetworkDetailsSpec = @{
+         DnsName = "$DomainName-vc01.$domain"
+      }
+   }
+   ComputeSpec = @{
+      ClusterSpecs = @(
+         @{
+            DatastoreSpec = @{
+               VSanDatastoreSpec = @{
+                  DatastoreName = "$DomainName-ds01"
+               }
+            }
+            Name = "$DomainName-cl01"
+            NetworkSpec = @{
+               VdsSpecs = @(
+                  @{
+                     Name = "$DomainName-vds01"
+                     PortGroups = @(
+                        @{
+                           Name = 'management'
+                           TransportType = 'MANAGEMENT'
+                        }
+                     )
+                  }
+                  @{
+                     Name = "$DomainName-vds02"
+                  }
+               )
+            }
+         }
+      )
+      NsxTSpec = @{
+         NSXManagerAdminPassword = "VMware123!123"
+         VipFqdn = "$domainName-nsx01.$domain"
+      }
+   }
+}
+
+$EdgeClusterParams = @{
+   EdgeClusterName = "$($domainSpec.ComputeSpec.ClusterSpecs[0].Name)-ec01"
+}
+
+############################################################################################################################
+# Enable WCP
+############################################################################################################################
+
+# --------------------------------------------------------------------------------------------------------------------------
+# WCP enablement parameters
+$ContentLibraryParams = @{
+   Name = "$DomainName-lib01"
+   Url = 'https://wp-content.vmware.com/v2/latest/lib.json'
+   SslThumbprint = 'AD:DA:3D:B9:99:75:1D:FF:2E:28:CA:92:83:64:38:20:5B:55:94:DC'
+   DownloadOnDemand = $true
+   Datastore = $domainSpec.ComputeSpec.ClusterSpecs[0].DatastoreSpec.VSanDatastoreSpec.DatastoreName
+}
+$DeepLearningVMContentLibraryParams = @{
+   Name = "$DomainName-lib02"
+   Url = 'https://packages.vmware.com/dl-vm/lib.json'
+   SslThumbprint = 'AD:DA:3D:B9:99:75:1D:FF:2E:28:CA:92:83:64:38:20:5B:55:94:DC'
+   DownloadOnDemand = $true
+   Datastore = $domainSpec.ComputeSpec.ClusterSpecs[0].DatastoreSpec.VSanDatastoreSpec.DatastoreName
+}
+
+$ClusterName = $domainSpec.ComputeSpec.ClusterSpecs[0].Name
+
+$WcpClusterParams = @{
+   SizeHint = 'Small'
+   ManagementNetwork = @{
+      Name = (
+         $domainSpec.ComputeSpec.ClusterSpecs[0].
+            NetworkSpec.VdsSpecs[0].
```

Comment:
It looks like the distributed switches have some roles associated to them - first is for management and the other for workload networks? If so, document this in some way.